scripts/prepare-unity.sh

```diff
@@ -37,11 +37,22 @@ export GOPATH
 
 if [ -d "$IMPROBABLE_UNITY_VERSIONED" ]; then
   echo "Unity is already installed in:" $IMPROBABLE_UNITY_VERSIONED
-  exit 0
 else
   echo "Could not find a Unity $UNITY_PINNED_VERSION installation."
 fi
 
+if [ -d "$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines/LinuxStandaloneSupport" ] || \
+   [ -d "$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines/AndroidPlayer" ] || \
+   [ -d "$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines/iOSSupport" ] || \
+   [ -d "$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines/WindowsStandaloneSupport" ] || \
+   [ -d "$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines/MacStandaloneSupport" ]
```

Comment:
For Mac this path is different for MacStandaloneSupport, also the directories should be:

`"$IMPROBABLE_UNITY_VERSIONED/Editor/Data/PlaybackEngines/..."`

Reply:
yes you're right - the paths here are the mac ones

will add the windows/mac changes 👍